test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -780,7 +780,6 @@ void overwriteUncleHeaderForTest(mObject& uncleHeaderObj, TestBlock& uncle, std:
         {
             uncleHeader.setTimestamp(importedBlocks.at(number).blockHeader().timestamp() + 1);
             sealEngine->populateFromParent(uncleHeader, importedBlocks.at(number).blockHeader());
-            uncleHeader.setAuthor(h160::random()); // make each populated block unique
```

Comment:
Sorry. Added it back.

But do you really want to generate different test each time you run the generator?

Reply:
hmm. it is different anyway because build info is updated. but you right maybe a better solution would be to use determenistic addresses. like 10000..01, 10000...02

Reply:
test filling does not work after your changes

Reply:
no it is not related to timestamp issue. it is some other issue. it happens some time for some reason. but other time test filling works. strange

Reply:
so test is different all times anyway. timestamp is different when you regenerate the test. so blockhash would be different. different uncle author or static - the hash would be different anyway.